### PR TITLE
Fix issue #313 (different results with Nan's and Zeros in power series)

### PR DESCRIPTION
**Pull request by martin-springer**
- [x] Code changes are covered by tests
- [x] Code changes have been evaluated for compatibility/integration with TrendAnalysis
- [ ] ~New functions added to `__init__.py`~
- [ ] ~API.rst is up to date, along with other sphinx docs pages~
- [ ] ~Example notebooks are rerun and differences in results scrutinized~
- [x] Updated changelog

### CODE REVIEW

File: rdtools/utilities.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+"""Utility functions for rdtools."""
+
+
+def robust_quantile(x, q):
+    """
+    Compute the q-th quantile of a time series (x), ignoring small values and NaN's.
+    NaN's and small values [x < Q(x,q)/1000] are removed before calculating the quantile.
+    This function ensures that time series with NaN's and distributions without
+    NaN's return the same results.
```
Comment (mdeceglie) [inline rdtools/utilities.py:9]:
```suggestion
    NaN's return the same results. Should only be used if x is expected to be ≥0.
```

File: rdtools/utilities.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    return q
+
+
+def robust_median(x, q=0.99):
+    """
+    Compute the median of a time series (x), ignoring small values and NaN's.
+    NaN's and small values [Q(x,q)/1000] are removed before calculating the mean.
+    This function ensures that time series with NaN's and distributions without
+    NaN's return the same results.
```
Comment (mdeceglie) [inline rdtools/utilities.py:35]:
```suggestion
    NaN's return the same results.  Should only be used if x is expected to be ≥0.
```

File: rdtools/utilities.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+    return mdn
+
+
+def robust_mean(x, q=0.99):
+    """
+    Compute the mean of a time series (x), ignoring small values and NaN's.
+    NaN's and small values [x < Q(x,q)/1000] are removed before calculating the mean.
+    This function ensures that time series with NaN's and distributions without
+    NaN's return the same results.
```
Comment (mdeceglie) [inline rdtools/utilities.py:61]:
```suggestion
    NaN's return the same results.  Should only be used if x is expected to be ≥0.
```

**Review by mdeceglie — COMMENTED** [review]
Looks good. I suggest we make a minor change to the docstrings to remind us and users about the x≥0 assumption

**Comment by martin-springer** [comment]
> Looks good. I suggest we make a minor change to the docstrings to remind us and users about the x≥0 assumption

@mdeceglie - Yes, that sounds good to me. I committed your additions.

**Review by mdeceglie — APPROVED** [review]

**Pull request merged**